Neos.Flow/Classes/Validation/Validator/UniqueEntityValidator.php

```diff
@@ -57,32 +57,32 @@ class UniqueEntityValidator extends AbstractValidator
     protected function isValid($value)
     {
         if (!is_object($value)) {
-            throw new InvalidValidationOptionsException('The value supplied for the UniqueEntityValidator must be an object.', 1358454270);
+            throw new InvalidValidationOptionsException('The value supplied for the UniqueEntityValidator must be an object.', 1_358_454_270);
         }
 
         $classSchema = $this->reflectionService->getClassSchema(TypeHandling::getTypeForValue($value));
         if ($classSchema === null || $classSchema->getModelType() !== ClassSchema::MODELTYPE_ENTITY) {
-            throw new InvalidValidationOptionsException('The object supplied for the UniqueEntityValidator must be an entity.', 1358454284);
+            throw new InvalidValidationOptionsException('The object supplied for the UniqueEntityValidator must be an entity.', 1_358_454_284);
         }
 
         if ($this->options['identityProperties'] !== null) {
             $identityProperties = $this->options['identityProperties'];
             foreach ($identityProperties as $propertyName) {
                 if ($classSchema->hasProperty($propertyName) === false) {
-                    throw new InvalidValidationOptionsException(sprintf('The custom identity property name "%s" supplied for the UniqueEntityValidator does not exists in "%s".', $propertyName, $classSchema->getClassName()), 1358960500);
+                    throw new InvalidValidationOptionsException(sprintf('The custom identity property name "%s" supplied for the UniqueEntityValidator does not exists in "%s".', $propertyName, $classSchema->getClassName()), 1_358_960_500);
                 }
             }
         } else {
             $identityProperties = array_keys($classSchema->getIdentityProperties());
         }
 
-        if (count($identityProperties) === 0) {
-            throw new InvalidValidationOptionsException('The object supplied for the UniqueEntityValidator must have at least one identity property.', 1358459831);
+        if ((is_countable($identityProperties) ? count($identityProperties) : 0) === 0) {
```

Comment:
it seesm not to contextualise (no surprise) but this is eg. an unnecessary addition....